desec.py

```diff
@@ -1409,6 +1444,47 @@ def tlsa_record(
     return f"{int(usage)} {int(selector)} {int(match_type)} {hex_data}"
 
 
+class CliClientFormatter(logging.Formatter):
+    """Pretty prints requests and response logs for CLI usage."""
+
+    def format(self, record: logging.LogRecord) -> str:
+        """Format a log record for cli usage.
+
+        Args:
+            record: Log record to format.
+
+        Returns:
+            Formatted log record as string.
+        """
+        message = record.getMessage()
+        if getattr(record, "params", None):
+            message += "\nParams:"
+            for k, v in record.params.items():
+                message += f"{k}: {v}"
+        if getattr(record, "body", None):
+            message += "\nBody:\n"
+            message += json.dumps(record.body, indent=2)
+        if getattr(record, "response_body", None):
+            message += "\n"
+            message += json.dumps(record.response_body, indent=2)
+            message += "\n"
+        return message
+
+
+def configure_cli_logging(level: int):
+    """Set up logging configuration when using the module as a command-line interface.
+
+    Args:
+        level: Logging level to set for desec.client logger.
+    """
+    http_handler = logging.StreamHandler(stream=sys.stdout)
```

Comment:
Wouldn't it make more sense to send debugging and status information to stderr? That'd make it easier to separate it from the regular output, if needed.

Reply:
Done